webapp/src/main/java/org/opentestsystem/rdw/admin/service/StudentGroupBatchService.java

```diff
@@ -10,7 +10,7 @@
 /**
  * Service responsible for managing {@link StudentGroupBatch}s
  */
-//TODO: adjust permissions appropriately
+@PreAuthorize("hasAuthority('PERM_GROUP_WRITE')")
```

Comment:
Changed this service to require GROUP_WRITE permissions and moved the annotation to the class level.